src/_playground/documentation/Playground/Playground.js

```diff
@@ -179,6 +179,7 @@ export class Playground extends Component {
                               <select
                                   className='form-control'
                                   name={item.attribute}
+                                  onBlur={this.updateComponent}
```

Comment:
dumb q: does this double fire because it is on both onBlur and onChange?

Reply:
I thought it would too, but I played around with it and everything seems fine.